korangar/src/interface/linked.rs

```diff
@@ -94,9 +97,11 @@ mod test {
     fn links_can_be_hidden() {
         let element = LinkedElement::new();
 
+        #[cfg(feature = "debug")]
```

Comment:
The test tests `link_hidden()`! So the whole test should get the cfg!

Reply:
Good catch!